prac_05/wimbledon.py

```diff
@@ -0,0 +1,47 @@
+"""
+Wimbledon
+Estimate: 25 minutes
+Actual: 31 minutes
+"""
+
+FILENAME = "wimbledon.csv"
+
+
+def main():
+    """Prints Wimbledon winners, and country of Wimbledon winners"""
+    lines = extract_data(FILENAME)
+    player_to_wins, winning_countries = extract_player_country_results(lines)
+    print_results(player_to_wins, winning_countries)
+
+
+def extract_data(filename):
+    """Extract data from input file"""
+    with open(filename, 'r', encoding="utf-8-sig") as in_file:
+        in_file.readline()  # Remove data structure line
+        lines = [line.split(',') for line in in_file]
+        return lines
+
+
+def extract_player_country_results(lines):
+    """Extract set of winning players countries, and dictionary of winners names and number of wins"""
+    yearly_winning_players = [line[2] for line in lines]
```

Comment:
Consider changing "Line[2]" to have its own constant that describes what it does. Something like: WINNER_COLUMN